code/style.css

```diff
@@ -1,4 +1,209 @@
+
+  /* color */
+  --color-black: #000000ef;
+  --color-white: #f5f5f5;
+  --color-accent: #bf3019;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Lato", sans-serif;
+  background-color: var(--color-black);
+  color: var(--color-white);
+}
+/* h2, h3 reserved for title */
+h2,
+h3 {
+  font-family: "Libre Baskerville", serif;
+  font-weight: var(--fontweight-baskerville-bold);
+  cursor: default;
+}
+h2 {
+  line-height: 2.25rem;
+}
+a {
+  text-decoration: none;
+  color: inherit;
+}
+p {
+  cursor: default;
+}
+.header {
+  max-width: 62rem;
+  margin: 0 auto;
+  height: 4rem;
+  display: flex;
+  padding: 2em 1em;
+  justify-content: space-between;
+  align-items: center;
+}
+.branding-tablet,
+.branding {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.branding-tablet {
+  display: none;
+}
+.branding-logo {
+  display: inline-block;
+  width: 1.25rem;
+  height: 1.25rem;
+  margin-right: 0.5rem;
+  border-radius: 50%;
+  background-color: var(--color-accent);
+}
+
+.branding-title {
+  display: inline;
+  font-family: "Luxurious Roman", cursive;
+  font-size: 1.25rem;
+  cursor: default;
+}
+
+.nav {
+  flex: 1;
+  justify-content: space-between;
+  align-items: center;
+  display: none;
+}
+.nav-menu {
+  padding: 0.5em 1em;
+  margin: 0.15rem;
+}
+.nav-menu:hover {
+  color: var(--color-accent);
+}
+.big-news,
+.popular {
+  max-width: 55rem;
+  margin: 0.5rem auto;
+}
+
+.big-news {
+  display: grid;
+  gap: 2rem;
+  grid-template-columns: repeat(auto-fill, minmax(18rem, 1fr));
+  margin-bottom: 4rem;
+  margin-top: 3rem;
+  padding: 0 1em;
+}
+
+.big-news-description {
+  display: flex;
+  flex-direction: column;
+}
+.description-date {
+  margin-top: 0;
+}
+.big-news-img {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+.card-date,
+.description-date {
+  font-size: var(--fontsize-small);
+  font-weight: var(--fontweight-lato-light);
+}
+
+.card-date {
+  margin-top: 0;
+}
+.description-summary {
+  font-weight: var(--fontweight-lato-light);
+  line-height: 1.25rem;
+}
+
+.popular {
+  background-color: var(--color-white);
+  color: var(--color-black);
+  padding: 0 1em;
+}
+.popular-heading {
+  display: flex;
+  align-items: center;
+}
+.popular-title {
+  color: var(--color-accent);
+  text-transform: uppercase;
+  letter-spacing: 0.2rem;
+  margin-right: 1rem;
+}
+.popular-divider {
+  flex: 1 1;
+  background-color: var(--color-accent);
+  height: 1px;
+  width: 100%;
+}
+.popular-card-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(18rem, 1fr));
+  padding-bottom: 2em;
+  gap: 0.75rem;
+}
+.popular-card {
+  padding: 1em;
+  -webkit-box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.56);
+  -moz-box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.56);
+  box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.56);
+  transition: all 0.25s ease-in;
+}
+.popular-card:hover {
+  transform: scale(1.01);
```

Comment:
I really like these effects put to your popular cards. That slow ease in when you mouse over is a really nice, professional looking touch! 😃

Reply:
> I really like these effects put to your popular cards. That slow ease in when you mouse over is a really nice, professional looking touch! 😃

I'm using this site for box shadow settings where I can just adjust slides and copy the result code into css!  Here is for your reference :) 
https://cssgenerator.org/box-shadow-css-generator.html

Reply:
That's awesome! Bookmarking it straight away, thanks! :)